index.Rmd

```diff
@@ -515,8 +527,33 @@ plot(forest, col = "dark green", legend = FALSE)
 
 # Today's summary
 
-We learned about:
+Today you performed a supervised classification, you identified patches and sieve connected cells, and you learned to deal with thematic raster data. Some functions to retain: 
+
+## Data exploration 
+
+* `hist()`: Create a histogram for each layer of a `SpatRaster`.
+* `pairs()`: Create a scatterplot for each pair of layers of a `SpatRaster`. 
+* `app()`: Apply a function to all pixels of a `SpatRaster` more efficiently. 
+
+## Training data preparation 
+
+* `extract()`: Retrieve a value for the raster below a polygon. 
+
+## Contruct a Random Forest model
+
+* `ranger()`: Build a Random Forest model based on a data frame of predictors and a response variable. `importance = "permutation"` and `importance()` to get the statistical importance of each predictor. 
+
+## Run a model on the data
+
+* `predict()`: Predict raster values based on a predefined model. 
+
+## Applying a raster sieve by identifying patches 
+
+* `setValues()`: Assign a new value to a raster
+* `patches()`: Identify patches of raster cells and attribute an ID to each. Specify neighbors with `direction = 4`or `direction = 8`. 
+* `freq()`: Count the number of raster cells per ID. 
+
+## Working with thematic rasters
+
+* `segregate()`: Create a `SpatRaster` object for each layer or class. 
```

Comment:
Added a summary of the most important functions to help students process this lengthy tutorial. Took inspiration from the tutorial Intro to Raster.